6starter.tex

```diff
@@ -210,38 +210,38 @@
 
  \hline
 %  \pagebreak
- 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (aka spawning biomass, aka B). The calculated values are reported to the SD report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
+ 1 & Depletion basis: & \multirow{1}{1cm}[-0.25cm]{\parbox{12.5cm}{Selects the basis for the denominator when calculating degree of depletion in reproductive output (a.k.a., \gls{sb}, a.k.a. B). The calculated values are reported to the \gls{sd} report relative to a fraction, X, of a comparable quantity calculated in the benchmark section or elsewhere.}} \Tstrut\\
    & & \\
    & & \\
    & 0 = skip; & \Tstrut \\
-   & 1 = X*B0; & Relative to virgin spawning biomass. \\
-   & 2 = X*B\textsubscript{MSY}; & Relative to spawning biomass that achieves MSY. \\
-   & 3 = X*B\textsubscript{styr}; and & Relative to model start year spawning biomass. \\
-   & 4 = X*B\textsubscript{endyr}. & Relative to spawning biomass in the model end year. \\
-   & 5 = X*Dynamic B0 & Relative to the calculated dynamic B0. \\
+   & 1 = $X*B0$; & Relative to virgin spawning biomass. \\
+   & 2 = $X*B_{MSY}$; & Relative to spawning biomass that achieves \gls{msy}. \\
+   & 3 = $X*B_{styr}$; and & Relative to model start year spawning biomass. \\
+   & 4 = $X*B_{endyr}$. & Relative to spawning biomass in the model end year. \\
+   & 5 = $X*Dynamic B0$ & Relative to the calculated dynamic B0. \\
```

Comment:
I do not think that the space actually renders in the equation. I believe you need a tilde to enforce the space.